classes/NMLClock.sc

```diff
@@ -160,7 +160,7 @@ BeaconClock : TempoClock {
 				count = msg[2];
 				numReplies = msg[3];
 				onlinePeers = addrBook.onlinePeers;
-				if((name != addrBook.me.name && numReplies == onlinePeers.size) || (numReplies < 3 && (numReplies > 0) && numReplies == onlinePeers.size), { // ignore my own beacons if possible
+				if(((name != addrBook.me.name) && (numReplies == (onlinePeers.size-1))) || ((numReplies < 3) && (numReplies > 0) && (numReplies == onlinePeers.size)), { // ignore my own beacons if possible
```

Comment:
just a note: this is a rather inefficient way of writing it. Maybe better, because this is a method that is called often:

```supercollider
if(
((name != addrBook.me.name) and: { numReplies == (onlinePeers.size-1) }) 
     or: { 
       (numReplies < 3) 
       and: { numReplies > 0 } 
       and: { numReplies == onlinePeers.size }  
      }
),
```